vmse2000.py

```diff
@@ -330,6 +340,18 @@ def _init_socket(self):
         else:
             print("No UDP socket")
 
+    def store_evidence(self, profanity):
+        if self.evidence_file:
+            timestamp = time.strftime("%Y-%m-%dT%H:%M:%S")
+            try:
+                with open(self.evidence_file, "a") as f:
+                    writer = csv.writer(f)
```

Comment:
how about jsonl (newline-separated json) instead of csv? defined encoding and well-defined parsing rules: all the things csv doesn't have :)

Reply:
Yeah, fine with me. Gives us more flexibility.